fuzz/fuzz_helper/src/lib.rs

```diff
@@ -146,15 +146,39 @@ pub fn process_ctap_any_type(data: &[u8]) -> arbitrary::Result<()> {
     Ok(())
 }
 
+fn setup_env(
+    unstructured: &mut Unstructured,
+    state: &mut CtapState,
+    env: &mut TestEnv,
+) -> arbitrary::Result<()> {
+    if bool::arbitrary(unstructured)? {
+        test_helpers::env::setup_enterprise_attestation(state, env).ok();
```

Comment:
I think the problem is that we currently only fuzz for panics and not for correctness, so I see why we don't care about errors. Let's keep it like that for now and hope we rewrite everything when/if we start checking for correctness, because for sure we will miss those silent errors.